api/vendor/google/apiclient-services/src/Google/Service/Compute/HttpHealthCheck.php

```diff
@@ -0,0 +1,138 @@
+<?php
```

Comment:
@jalmiShreya @sharangsamonkar - can we verify why these vendor files are added for?